DigitalSolutionsStudio.Api/D-Studio. Test task C# code sample.cs

```diff
@@ -0,0 +1,121 @@
+﻿private void btnGetData_Click(object sender, EventArgs e)
+{
+    // clear previous request data
+    string sRequest = "";
+    lstData.Items.Clear();
+
+    // format request string based upon user input
+    if (cboHistoryType.Text.Equals("Tick Datapoints"))
+    {
+        // request in the format:
+        // HTX,SYMBOL,NUMDATAPOINTS,DIRECTION,REQUESTID,DATAPOINTSPERSEND<CR><LF>
+        sRequest = String.Format("HTX,{0},{1},{2},{3},{4}\r\n", txtSymbol.Text, txtDatapoints.Text, txtDirection.Text, txtRequestID.Text, txtDatapointsPerSend.Text);
```

Comment:
Such code repetition should be avoided by creating a method with 'params' variable for string formatting.